java/clients/src/test/java/sleeper/clients/admin/InstancePropertyReportTest.java

```diff
@@ -58,19 +58,24 @@ public void shouldPrintInstancePropertyReportWhenChosen() {
                 .containsAnyOf(Stream.of(SystemDefinedInstanceProperty.values())
                         .map(SystemDefinedInstanceProperty::getPropertyName)
                         .toArray(String[]::new))
-                // Then check some set property values are present in the output
-                .contains("sleeper.account: 1234567890\n")
-                .contains("sleeper.log.retention.days: 1\n")
-                .contains("sleeper.tags: name,abc,project,test\n")
-                .contains("sleeper.vpc: aVPC\n");
+                // Then check some set property values and their descriptions are present in the output
+                .contains("# The AWS account number. This is the AWS account that the instance will be deployed to\n" +
+                        "sleeper.account: 1234567890\n")
+                .contains("# The length of time in days that CloudWatch logs are retained\n" +
+                        "sleeper.log.retention.days: 1\n")
+                .contains("# A list of tags for the project\n" +
+                        "sleeper.tags: name,abc,project,test\n")
+                .contains("# The id of the VPC to deploy to\n" +
+                        "sleeper.vpc: aVPC\n")
+                .contains("# The S3 bucket name used to store configuration files.\n" +
+                        "sleeper.config.bucket: sleeper-test-instance-config\n");
```

Comment:
Can we assert a few properties that display together as a single string, with the beginnings of the lines lined up? It'd be good to see if there's any whitespace between properties, or if we need any.

Can we also assert on a property with a description over multiple lines, or a particularly long description?